docs/source/quick_start.rst

```diff
@@ -0,0 +1,193 @@
+============
+Quick Start
+============
+
+  This guide is intended to get new Kokkos users started in less than one hour.  Kokkos Core is the foundation for all other Kokkos Ecosystem projects.
+
+------------
+Requirements
+------------
+
+::
+
+  CMake >=3.18
+  gcc >= 8.3.1
+  CUDA >= 11
+  rocm >=5.7
+
+--------------------------------------
+Download Current Release, Set Up Build 
+--------------------------------------
+
+..
+  Choice 1 for obtaining current release:
+..
+
+* `Click to Download Current Release <https://github.com/kokkos/kokkos/archive/refs/heads/master.zip>`_ 
```

Comment:
This renders weirdly for me: 
<img width="821" alt="Screenshot 2024-02-19 at 5 01 55 PM" src="https://github.com/kokkos/kokkos-core-wiki/assets/6853627/a5b6ec80-3b0c-4d73-abbf-9ac506ded5f9">